web/TrackPlayer/PlaylistPlayer.ts

```diff
@@ -20,28 +20,30 @@ export class PlaylistPlayer extends Player {
   }
 
   protected async onTrackEnded() {
-    switch (this.repeatMode) {
-      case RepeatMode.Track:
-        if (this.currentIndex !== undefined) {
-          await this.goToIndex(this.currentIndex);
-        }
-        break;
-      case RepeatMode.Playlist:
-        if (this.currentIndex === this.playlist.length - 1) {
-          await this.goToIndex(0);
-        }
-        break;
-      default:
-        try {
-          await this.skipToNext();
-        } catch (err) {
-          if ((err as Error).message !== 'playlist_exhausted') {
-            throw err;
+    try {
```

Comment:
Can you please walk me through the motivation of this change?

Reply:
This got mixed up with my other pull request, It's a bug fix for web.

Reply:
@Bram-dc cool, can you force push this PR and remove that commit?

Reply:
✅